snippets/comment.liquid

```diff
@@ -5,7 +5,7 @@
 
   <em>
     {% capture author %}<strong>{{ comment.author }}</strong>{% endcapture %}
-    {% capture date %}<strong><time datetime="{{ comment.created_at | date: 'yyyy-MM-dd' }}">{{ comment.created_at | date: format: 'month_day_year' }}</time></strong>{% endcapture %}
+    {% capture date %}<strong><time datetime="{{ comment.created_at | date: 'yyyy-MM-dd' }}">{{ comment.created_at | date: 'month_day_year' }}</time></strong>{% endcapture %}
```

Comment:
is that a bug in the main repo? If yes, submit a fix PR

Reply:
Reverted. Missed during bulk replace in visual studio :)